pom.xml

```diff
@@ -21,7 +21,7 @@
     <modelVersion>4.0.0</modelVersion>
     <groupId>org.ballerinalang</groupId>
     <artifactId>connector-cassandra-parent</artifactId>
-    <version>0.8.6</version>
+    <version>0.9.0</version>
```

Comment:
Shall we change the version to `0.99.0` please. Since the current released version is `0.8.6`, we need to keep `0.9.0` for 1.2.x release path

Reply:
Updated in bf1bea8